src/App.js

```diff
@@ -1,57 +1,39 @@
 import React from "react";
-import logo from "./logo.svg";
-import { Counter } from "./store/counter/Counter";
-import "./App.css";
+import { Switch, Route } from "react-router-dom";
+import about from "./pages/about";
+import books from "./pages/books";
+import error from "./pages/error";
+import favourites from "./pages/favourites";
+import home from "./pages/home";
+import profile from "./pages/profile";
+import signin from "./pages/signin";
+import signup from "./pages/signup";
```

Comment:
Let's change all of those to Captial letter 
ex `About, Home, Books, Error, Signin`
If we have a component like a footer and it is called like this it will cause issues, since the compiler will think this is the HTML tag. So make components start with Capital letters always.